packages/shared/src/components/onboarding/common.ts

```diff
@@ -22,3 +22,7 @@ export interface OnboardingStepProps {
   onClose?: (e: React.MouseEvent | React.KeyboardEvent) => void;
   isModal?: boolean;
 }
+
+export const requiredTagsThreshold = 5;
```

Comment:
Personally prefer capital version with snake case for magic number solution, but no hard feelings about it.